LICENSE

```diff
@@ -1,19 +1,20 @@
-Copyright 2018 MYOB Technology Pty Ltd
+The MIT License (MIT)
```

Comment:
Noticed the previous version was 2019, while this one is still 2013. Did we need to replace the LICENSE file?

Reply:
Modified file to 2020 version